public/index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <title>Menu Service Component</title>
+  <link href="https://fonts.googleapis.com/css?family=DM+Sans|Open+Sans|Questrial&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div>
+    <div id="app"></div>
+  </div>
+  <script type="text/javascript" src="./bundle.js"></script>
+  <link rel="stylesheet" type="text/css" href="styles.css">
```

Comment:
This should go in the head (the line importing CSS styles).

Reply:
I'm not sure either... but I think you're probably right here.